public/index.html

```diff
@@ -1,10 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#325">
-    <link rel="shortcut icon" href="https://vega.github.io/favicon.ico"/>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="theme-color" content="#325" />
+    <meta name="keywords" content="vega, editor, vega-editor" />
+    <meta
+      property="og:description"
+      content="The Vega editor is a web application for authoring and testing Vega and Vega-Lite visualizations. It includes a number of example specifications that showcase both the visual encodings and interaction techniques."
+    />
+    <meta property="og:title" content="Editor/IDE for Vega and Vega-Lite" />
+    <meta property="og:url" content="https://vega.github.io/editor/" />
+    <meta property="og:image" content="https://vega.github.io/favicon.ico" />
```

Comment:
We probably need a better teaser image, no?

Reply:
Yeah, I thought so. But we don't have our logo hosted anywhere, right?

Reply:
@domoritz How will we test it though? All these metadata.

Reply:
You mean https://github.com/vega/logos?